app/javascript/test/game.test.js

```diff
@@ -1,24 +1,53 @@
 import React from 'react';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { mount } from 'enzyme';
+
 import './setupTests';
-import { shallow, mount, render } from 'enzyme';
 
 import Game from '../components/game/Game';
 
 describe('Game', () => {
   let wrapper;
-
+  let mock;
   beforeEach(() => {
-    wrapper = shallow(<Game />)
+    wrapper = mount(<Game />)
+    mock = new MockAdapter(axios);
+
   })
 
     test('render a h1 welcome message', () => {
       expect(wrapper.find("h1").text()).toContain("Tic-Tac-Toe");
     });
-    test('render a count of html tags on the game component', () => {
-      expect(wrapper.find('div').length).toBe(3);
-      expect(wrapper.find('h1').length).toBe(1);
-      expect(wrapper.find('ul').length).toBe(1);
-      expect(wrapper.find('li').length).toBe(9);
 
+    test('that a button with a new game is preseent', () => {
+      wrapper.find('button').simulate('submit');
```

Comment:
Since this test is only verifying that a button exists, you probably don't need to stimulate a submit event here.